src/index.rs

```diff
@@ -355,3 +368,34 @@ impl Index {
         Ok(Query { inner: query })
     }
 }
+
+impl Index {
+    fn register_custom_text_analyzers(index: &tv::Index) {
+        let mut analyzers = HashMap::new();
+        analyzers.insert("ar_stem", Language::Arabic);
+        analyzers.insert("da_stem", Language::Danish);
+        analyzers.insert("nl_stem", Language::Dutch);
+        analyzers.insert("fi_stem", Language::Finnish);
+        analyzers.insert("fr_stem", Language::French);
+        analyzers.insert("de_stem", Language::German);
+        analyzers.insert("el_stem", Language::Greek);
+        analyzers.insert("hu_stem", Language::Hungarian);
+        analyzers.insert("it_stem", Language::Italian);
+        analyzers.insert("no_stem", Language::Norwegian);
+        analyzers.insert("pt_stem", Language::Portuguese);
+        analyzers.insert("ro_stem", Language::Romanian);
+        analyzers.insert("ru_stem", Language::Russian);
+        analyzers.insert("es_stem", Language::Spanish);
+        analyzers.insert("sv_stem", Language::Swedish);
+        analyzers.insert("ta_stem", Language::Tamil);
+        analyzers.insert("tr_stem", Language::Turkish);
```

Comment:
This doesn't need to be a hashmap here, an array would be fine.

Reply:
Done